pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, trainer: 'pl.Trainer', log_gpu_memory: Optional[str] = None)
 
     def on_trainer_init(
         self,
-        logger: LightningLoggerBase,
+        logger: Union[bool, Iterable[LightningLoggerBase], LightningLoggerBase],
```

Comment:
don't we want to put the logger base before iterable?

Reply:
Does this have an impact or just for looks? In any case, I agree

Reply:
for readability I guess, one could argument that ordering primitives first then composed types is easier to read.
just a nit

Reply:
Definitely a nit. I wonder if it's possible to configure the code formatter/linter to do this? Much easier to enforce for consistency, which would go a long way in aiding readability.

Reply:
@awaelchli updated.